packages/manager/modules/pci/src/projects/project/instances/instance/stop/stop.controller.js

```diff
@@ -0,0 +1,73 @@
+import get from 'lodash/get';
+
+export default class PciInstanceStopController {
+  /* @ngInject */
+  constructor(
+    $q,
+    $translate,
+    Poller,
+    CucCloudMessage,
+    PciProjectsProjectInstanceService,
+    OvhApiCloudProjectInstance,
+  ) {
+    this.$q = $q;
+    this.$translate = $translate;
+    this.Poller = Poller;
+    this.CucCloudMessage = CucCloudMessage;
+    this.PciProjectsProjectInstanceService = PciProjectsProjectInstanceService;
+    this.OvhApiCloudProjectInstance = OvhApiCloudProjectInstance;
+  }
+
+  $onInit() {
+    this.isLoading = false;
+  }
+
+  waitingInstanceStop() {
```

Comment:
```suggestion
  waitForInstanceStop() {
```
as it's a function, it should refer to an action not a state

Reply:
Fixed